web/saml.go

```diff
@@ -48,6 +50,8 @@ func loginWithSaml(c *Context, w http.ResponseWriter, r *http.Request) {
 		relayProps["redirect_to"] = redirectTo
 	}
 
+	relayProps["isMobile"] = strconv.FormatBool(isMobile)
```

Comment:
Same as Scott's comment about a constant, but for all cases of the string "isMobile".

Reply:
Done